engines/director/debugger.cpp

```diff
@@ -521,6 +538,24 @@ bool Debugger::cmdDisasm(int argc, const char **argv) {
 						}
 					}
 				}
+				debugPrintf("  Factories:\n");
```

Comment:
This block looks to be a duplicate of the one at line 505. Could both blocks be moved into a reusable function?

Reply:
See above